audformat/core/conftest.py

```diff
@@ -0,0 +1,26 @@
+import os
+import tempfile
+
+import numpy as np
+import pytest
+
+import audeer
+import audiofile
+
+
+@pytest.fixture(scope='session', autouse=True)
```

Comment:
Mhh, but it seems to have the disadvantage that it runs it for every docstring example. I still wonder if it's possible to set it up so that it's only executed during the docstring examples or alternatively for specific docstring examples.

Reply:
Ah, setting it to `'package'` seems to do the trick.

Reply:
I changed to package, but for whatever reason the output of `audeer.path('a')` did not contain the tmp folder for `'session'` nor for `'package'`. So it seems in both cases we are not inside the tmp folder. Which is strange because before tests were failing exactly because of the switch to the tmp folder.

Reply:
Sorry, I'm lost. What do you mean by "output of audeer.path('a')"?

Reply:
Inside a test (`test_expand_file_path`) I added:

```python
print(audeer.path('a'))
assert False
```

in order to force it printing out in which folder the test is executed and to my surprise it wasn't in the tmp folder, but inside `/home/audeering.local/hwierstorf/git/audeering/audformat/a` in both cases.

Reply:
But it's anyway the behavior we would like to have, so maybe we don't need to worry any further here.

Reply:
When I add `audeer.path('a')` in the docstring I get something like `/tmp/tmpacgu3lml/a` so the tmp folder seems indeed only be used there.